Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 10.4.0
 - [fixed] Fix an issue that stops some performance optimization being applied.
```

Comment:
I couldn't find an associated issue, not sure if PR #10579 should be added for context. Ignore if unnecessary, just pointing it out in case.

Reply:
Thanks @andrewheard , good catch. Updated.